kimshinah/src/components/BookingSection.tsx

```diff
@@ -0,0 +1,116 @@
+import styled from "styled-components";
+import { useState, ChangeEvent, useEffect } from "react";
+import Seating from "../components/Seating";
+import SeatingOccupied from "../components/SeatingOccupied";
+import Router from "next/router";
+
+const BookingSection = () => {
+  //any가 왜있어야 하는걸까...
+  const seatingArray: any = [];
```

Comment:
any 사용은 지양해주셔야해요 ~

```suggestion
  const seatingArray: number[] = [];
```

어떤 형식의 배열인지 타입을 추론할 수 없기 때문입니다~

Reply:
제가 type을 많이 써보질 않아서 무슨 타입을 배치를 해야할까 하다가 아무거나 number을 넣었는데 안되더라구요. 그래서 any를 넣은거였는데 제가 타입에 대해서 아직 지식이 많이 부족한 것 같습니다 ㅠㅠ 타입에 대해서 더 공부해 오겠습니다!!